elm.json

```diff
@@ -3,7 +3,7 @@
     "name": "QiTASC/hatchinq",
     "summary": "An experimental library for UI elements. Not for production use.",
     "license": "Apache-2.0",
-    "version": "12.0.0",
+    "version": "12.0.1",
```

Comment:
doesn't elm bump do this?

Reply:
I checked - you should leave this

Reply:
I didn't know about elm bump :D
When I run it, it will say 

> Based on your new API, this should be a MAJOR change (12.0.0 => 13.0.0)
Bail out of this command and run 'elm diff' for a full explanation.
Should I perform the update (12.0.0 => 13.0.0) in elm.json? [Y/n] 

So do we use `13.0.0` or do we keep `12.0.0` ?

Reply:
keep 12.0.0